package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+"""
+A PCA module for MDAnalysis
+==========================================================================
+
+:Authors: The pyPcazip Developers - Ardita Shkurti, Ramon Goni, Pau Andrio, Elena Breitmoser, Iain
+Bethune, Modesto Orozco, Charles Laughton
+:Year: 2015
+:Copyright: BSD
+
+
+The module contains code to perform Principal Component Analysis of trajectory
+data. For examples of the application of PCA to MD analysis see Wlodek et al.
+[Wlodek1997]_ and Sherer et al. [Sherer1999]_.
+
+Much of the code has been adapted from that in the `pyPcazip package`_.
+
+.. Rubric:: References
+
+..[Wlodek1997] Wlodek ST, Clark TW, Scott LR, McCammon JA (1997)
+               J Am Chem Soc 119, 9513-9522.
+
+..[Sherer1999] Sherer EC, Harris SA, Soliva R, Orozco M, Laughton CA (1999)
+               J Am Chem Soc 121, 5981-5991.
+
+.._`pyPcazip package`: https://bitbucket.org/ramonbsc/pypcazip
+
+
+
+.. examples
+
+Examples
+--------
+
+    import MDAnalysis as mda
+    from MDAnalysis.analysis.pca import Pca
```

Comment:
Actually, we have also a specific format with the projection values of in the PCA-space and this format is named pcz. But since in this case, we are integrating the basic version, we could rename the module as pca.